uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Service/DrupalUserService.php

```diff
@@ -51,6 +51,17 @@ public function createNew($email, $active = FALSE, $roles = []) {
     return $user;
   }
 
+  /**
+   * Checks if user record exist for email
+   *
+   * @param string $email
+   *
+   * @return object | boolean
```

Comment:
object or boolean? Should be object I think.

Reply:
If requested user is not found by email, response is a boolean.